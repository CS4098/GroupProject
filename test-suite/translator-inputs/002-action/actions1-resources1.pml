process test_action {
    action act {
	requires { a }
	provides { a }
    }
}
