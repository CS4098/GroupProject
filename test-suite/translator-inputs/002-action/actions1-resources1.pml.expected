bool a;

active proctype test_action()
{
	a ->
		a = true;
}